packages/api/src/router/user.ts

```diff
@@ -7,9 +7,12 @@ import { groupMembers, groupPermissions, groups, invites, users } from "@homarr/
 import { validation, z } from "@homarr/validation";
 
 import { createTRPCRouter, protectedProcedure, publicProcedure } from "../trpc";
+import { assertCredentialsEnabled } from "./invite/asserts";
 
 export const userRouter = createTRPCRouter({
   initUser: publicProcedure.input(validation.user.init).mutation(async ({ ctx, input }) => {
+    assertCredentialsEnabled();
```

Comment:
assert is a name that should only be used for test.
I suggest ``throwIfCredentialsDisabled()``

Reply:
I don't full agree. In my opinion it should be placed whereever we want to restrict the continous execution to something that's also why typescript code like this exists:

```ts
const assertNotNullable = <T>(input: T): asserts input is NonNullable<T> => {
  if (input === null || input === undefined) {
    throw new Error("Value is null or undefined");
  }
}
```

but as I don't use this assertion I'll rename it

Reply:
Okay, I see your point.
But regardless, ``throwIfXXXX()`` seems more descriptive than ``assertXXX`` in my opinion.
Up to you what name you want to keep